pkg/block/index.go

```diff
@@ -129,6 +132,16 @@ func (i HealthStats) OutOfOrderChunksErr() error {
 	return nil
 }
 
+// DoplicatedSampleErr is a function that checks for any duplicate samples in the HealthStats object.
```

Comment:
```suggestion
// DuplicatedSampleErr is a function that checks for any duplicate samples in the HealthStats object.
```

Reply:
what different?

Reply:
just some typos

Reply:
After I accepted the suggestion, there was a censorship anomaly.

Reply:
Thanks for the tip.